src/support/bytemuck.rs

```diff
@@ -0,0 +1,66 @@
+//! Support for the [`bytemuck`](https://crates.io/crates/bytemuck) crate.
+#![cfg(feature = "bytemuck")]
+#![cfg_attr(docsrs, doc(cfg(feature = "bytemuck")))]
+
+use bytemuck::Pod;
+use ruint::Uint;
+
+// Implement the `Pod` trait for `Uint` types with a size that is a multiple of 64, up to 1024.
+macro_rules! impl_pod {
+    ($($bits:expr),+ $(,)?) => {
+        $(
+            unsafe impl<const LIMBS: usize> bytemuck::Zeroable for Uint<{$bits}, LIMBS> {}
+            unsafe impl<const LIMBS: usize> Pod for Uint<{$bits}, LIMBS> where
```

Comment:
In other places (like aliases.rs) we let the macro run over (BITS, LIMBS) pairs. I think that's cleaner than making one argument generic.